repos/exawind/packages/openfast/package.py

```diff
@@ -14,6 +14,11 @@ class Openfast(bOpenfast):
     patch("segfault_message.patch", when="%clang@12.0.1 build_type=RelWithDebInfo")
     version("fsi", git="https://github.com/gantech/openfast.git", branch="f/br_fsi_2")
 
+    variant("rosco", default=True,
```

Comment:
```suggestion
    variant("rosco", default=False,
```
Let's keep the default off.  Otherwise it may trigger rebuilds and other issues for people not using ROSCO

Reply:
Fixed this! Makes sense.